main.nf

```diff
@@ -77,25 +81,44 @@ if (ch_tree_method == "iqtree") {
 //
 include { INPUT_CHECK                               } from './subworkflows/local/input_check'
 
+// New subworkflows for simplified mode (can also be used in full mode)
+if (params.workflow_mode == 'simplified') {
+    include { MCL_INFLATION_SELECTION               } from './subworkflows/local/mcl_inflation_selection'
+    include { INFER_TREES as INFER_SPECIES_TREES    } from './subworkflows/local/infer_trees'
+    include { INFER_TREES as INFER_REMAINING_TREES  } from './subworkflows/local/infer_trees'
+}
+
```

Comment:
Ahhhh, i see. That wasn't clicking with me until I saw this - so the simplified workflow necessarily hardcodes the methods used. So if we wanted a simplified and zoogle one, we'd just need to turn the last few modules for conservation core/analyses etc to a subworkflow and only do those steps when running in zoogle mode?